src/lib/input/input-container.scss

```diff
@@ -43,7 +42,7 @@ $mat-input-underline-disabled-background-image:
 // Global wrapper. We need to apply margin to the element for spacing, but
 // cannot apply it to the host element directly.
 .mat-input-wrapper {
-  margin: $mat-input-wrapper-spacing 0;
+  margin: 1em 0;
```

Comment:
After the changes I did below, it was only being used in one place which makes it pointless.